qubes-bug-report

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+import subprocess as sub
+import shlex
+import argparse
+import time
+import sys
+
+from os.path import expanduser
+
+#the term qube refers to a qubes vm
+
+def is_program_installed_in_qube( qube_name, command ):
+	is_installed = True
+
+	try:
+		shell_cmd = "qvm-run " + shlex.quote( qube_name ) + " --pass-io --no-color-output 'command -v " + command + "' &> /dev/null"
+		sub.check_call( shell_cmd, shell = True )
+
+	except sub.CalledProcessError:
+		is_installed = False
+
+	return is_installed
+
+
+#this function requires virsh 
+#domstate only works for Xen domU (guests)
+def is_qube_running( qube_name ):
+	runs = False
+
+	out = sub.check_output([ "virsh", "-c", "xen:///", "domstate", shlex.quote( qube_name ) ])
+	out = out.decode('utf-8').replace('\n', '')
+
+	if 'running' == out:
+		runs = True
+
+	return runs
+
+
+def get_qube_packages( qube_name ):
+	content = "## Qubes Packages\n\n"
+
+	#a qube can have more than one package manager installed (only one is functional)
+	pkg_cmd = { 'dpkg' : 'dpkg -l qubes-*', 'pacman' : 'pacman -Qs', 'rpm' : 'rpm -qa qubes-*' }
```

Comment:
Yes, it does:

~~~
% pacman -Qs qubes 
local/qubes-db-vm 3.1.2-3
    QubesDB libs and daemon service.
local/qubes-libvchan-xen 3.1.0-2
    The Qubes core libraries for installation inside a Qubes Dom0 and VM.
local/qubes-vm-core 3.1.12-2
    The Qubes core files for installation inside a Qubes VM.
local/qubes-vm-gui 3.1.4-5
    The Qubes GUI Agent for AppVMs
local/qubes-vm-utils 3.1.7-5
    Common Linux files for Qubes VM.
local/qubes-vm-xen 4.6.0-12
    Xen is a virtual machine monitor
~~~

but `qubes-*` does not work it should be just `qubes`